base/util.jl

```diff
@@ -400,7 +400,18 @@ printstyled(io::IO, msg...; bold::Bool=false, color::Union{Int,Symbol}=:normal)
 printstyled(msg...; bold::Bool=false, color::Union{Int,Symbol}=:normal) =
     printstyled(stdout, msg...; bold=bold, color=color)
 
-function julia_cmd(julia=joinpath(Sys.BINDIR::String, julia_exename()))
+"""
+    julia_cmd([julia]) :: Cmd
```

Comment:
In Base we still use the `sig -> return type` convention in docstrings

Reply:
Thanks, I fixed it.  (I thought I read somewhere that `:: type` is a way to go but I forgot where. I thought it makes sense to dedicate `->` to indicate returned object, like `copyto!(dest::AbstractArray, src) -> dest`, but I realized that this is not a common usage...)